netconfig.md

```diff
@@ -42,7 +42,7 @@ If you didn't click on `Allow access` when prompted by Windows, you may need to
 1. Type `Firewall` in your Windows task bar search box.
 2. Open the `Windows Defender Firewall` app from the search results.
 3. Click on `Advanced settings`.
-4. Right-click `Inbound Rules` and select `New Rule...`.
+4. Right-click `Outbound Rules` and select `New Rule...`.
```

Comment:
UDP port `7513` is the default **inbound** listening port of the Spacemesh node. Why do you think this should be changed?